examples/testdata/path_matcher/envoy_config.json

```diff
@@ -1,606 +0,0 @@
-{
```

Comment:
I would prefer to keep these files, maybe rename the directory to `route_match`. They show configs for snake-JSON variable bindings, autogenerated CORS, per-route configs with regex, etc. I find it helpful to reference.

Reply:
Done